protocol-units/bridge/integration-tests/tests/client_l1move_l2move.rs

```diff
@@ -0,0 +1,455 @@
+				HashLock(args.hash_lock.0),
+				BridgeAddress(args.initiator.clone()),
+				BridgeAddress(args.recipient.clone().into()),
+				Amount(AssetType::Moveth(args.amount)),
+			)
+			.await
+			.expect("Failed to lock bridge transfer");
+
+		let details = BridgeContract::get_bridge_transfer_details_counterparty(
+			&mut mvt_client_harness.movement_client,
+			BridgeTransferId(args.bridge_transfer_id.0),
+		)
+		.await
+		.expect("Failed to get bridge transfer details")
+		.expect("Expected to find bridge transfer details, but got None");
+
+		info!("Bridge transfer details: {:?}", details);
+
+		assert_eq!(details.state, 1, "Bridge transfer should be pending.");
+
+		sleep(Duration::from_secs(20)).await;
+		
+		let secret = b"secret";
+		let mut padded_secret = [0u8; 32];
+		padded_secret[..secret.len()].copy_from_slice(secret);
+
+		BridgeContract::abort_bridge_transfer(
+			&mut mvt_client_harness.movement_client,
+			BridgeTransferId(args.bridge_transfer_id.0)
+		)
+		.await
+		.expect("Failed to complete bridge transfer");
+
+		let details = BridgeContract::get_bridge_transfer_details_counterparty(
+			&mut mvt_client_harness.movement_client,
+			BridgeTransferId(args.bridge_transfer_id.0),
+		)
+		.await
+		.expect("Failed to get bridge transfer details")
+		.expect("Expected to find bridge transfer details, but got None");
+
+		assert_eq!(details.bridge_transfer_id.0, args.bridge_transfer_id.0);
+		assert_eq!(details.hash_lock.0, args.hash_lock.0);
+		assert_eq!(
+			&details.initiator_address.0,
+			&args.initiator,
+			"Initiator address does not match"
+		);
+		assert_eq!(details.recipient_address.0, args.recipient);
+		assert_eq!(details.amount.0, AssetType::Moveth(args.amount));
+		assert_eq!(details.state, 3, "Bridge transfer is supposed to be cancelled but it's not.");
+
+		Ok(())
+	}
+	.await;
+
+	test_result
+}
+#[tokio::test]
+async fn test_eth_client_should_build_and_fetch_accounts() {
+	let config = Config::default();
+	let (eth_client_harness, _config, _anvil) = TestHarness::new_only_eth(config).await;
+
+	let expected_accounts = [
+		address!("f39fd6e51aad88f6f4ce6ab8827279cfffb92266"),
+		address!("70997970c51812dc3a010c7d01b50e0d17dc79c8"),
+		address!("3c44cdddb6a900fa2b585dd299e03d12fa4293bc"),
+		address!("90f79bf6eb2c4f870365e785982e1f101e93b906"),
+		address!("15d34aaf54267db7d7c367839aaf71a00a2c6a65"),
+		address!("9965507d1a55bcc2695c58ba16fb37d819b0a4dc"),
+		address!("976ea74026e726554db657fa54763abd0c3a0aa9"),
+		address!("14dc79964da2c08b23698b3d3cc7ca32193d9955"),
+		address!("23618e81e3f5cdf7f54c3d65f7fbc0abf5b21e8f"),
+		address!("a0ee7a142d267c1f36714e4a8f75612f20a79720"),
+	];
+
+	let provider = eth_client_harness.rpc_provider().await;
+	let accounts = provider.get_accounts().await.expect("Failed to get accounts");
+	assert_eq!(accounts.len(), expected_accounts.len());
+
+	for (account, expected) in accounts.iter().zip(expected_accounts.iter()) {
+		assert_eq!(account, expected);
+	}
+}
```

Comment:
Remove this `test_eth_client_should_build_and_fetch_accounts` as it exists in the `eth_movement` integration test, we don't want to waste CI compute.

Reply:
Ignore this